bundles/mapping/mapwfs2/domain/WfsLayerModelBuilder.js

```diff
@@ -63,7 +63,12 @@ Oskari.clazz.define(
             defaultStyle.setLegend('');
 
             const mapModule = Oskari.getSandbox().findRegisteredModuleInstance('MainMapModule');
-            const wfsPlugin = mapModule.getLayerPlugins(layer.getLayerType());
+            let layerType = layer.getLayerType();
+            if (layerType === 'userlayer' || layerType === 'myplaces') {
+                layerType = 'wfs';
```

Comment:
This leaves me a bit cold to have references to userlayer and myplaces here. But I don't have a better suggestion for now so let's go with it and make it better in the future.